### PR TITLE
added documentation for mqtt 1.3 tutorial

### PR DESCRIPTION
**Pull request by arduionoGP**
Updated example from my post on the discourse site.

### CODE REVIEW

File: docs/software/other/mqtt.mdx [hunk 1]
```diff
@@ -380,3 +380,34 @@ if __name__ == '__main__':
     while client.loop() == 0:
         pass
 ```
+### Tutorial on using MQTT with version 1.3.46+
+
+Version 1.3.46+ is the first 1.3 release with MQTT and JSON fully functional on supported devices. Heltec JSON is unlikely to work reliably.
```
Comment (thebentern) [inline docs/software/other/mqtt.mdx:385]: In Heltec devices, JSON over MQTT

**Review by thebentern — CHANGES_REQUESTED** [review]

**Comment by thebentern** [comment]
Really great stuff! This is super informative to most folks getting started with MQTT

File: docs/software/other/mqtt.mdx (hunk 1 above)
Comment (arduionoGP) [reply docs/software/other/mqtt.mdx:385]: Absolutely. Thanks for the edit.  (I just got done testing the Heltec a few minutes ago and JSON over MQTT is inop but if it is turned off the protobuf method seems to work.)

**Review by sachaw — APPROVED** [review]
Would like some improved screenshots in the future, but good enough for now

**Comment by sachaw** [comment]
Actually, please remove the thumbs.db file

**Pull request merged**